tests/integration/istioctl/uninstall_test.go

```diff
@@ -0,0 +1,182 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istioctl
```

Comment:
this is not run in CI

Reply:
Move it to tests/integration/pilot/revisions maybe?

Reply:
Can we add that to CI? Though currently the setup is the same as tests/integration/pilot/revisions but it would be better to put it in a dedicated place for istioctl

Reply:
or maybe I can add it to tests/integration/pilot/revisions now to check, later I would put it to a more appropriate place and update test-infra correspondingly

Reply:
I care more about the efficiency of the tests than the names, so I would not like a new target. we can rename it if you really want